mesonbuild/compilers/mixins/elbrus.py

```diff
@@ -57,3 +59,14 @@ def get_program_dirs(self, env: 'Environment') -> typing.List[str]:
                 libstr = line.split(' ', 1)[1]
                 return [os.path.realpath(p) for p in libstr.split(':')]
         return []
+
+    def get_default_include_dirs(self) -> typing.List[str]:
+        os_env = os.environ.copy()
+        os_env['LC_ALL'] = 'C'
+        p = subprocess.Popen(self.exelist + ['-xc', '-E', '-v', '-'], env=os_env, stdin=subprocess.DEVNULL, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        stderr = p.stderr.read().decode('utf-8', errors='replace')
+        includes = []
+        for line in stderr.split('\n'):
+            if line.lstrip().startswith('--sys_include'):
+                includes.append(re.sub('\s*\\\\$', '', re.sub('^\s*--sys_include\s*', '', line)))
```

Comment:
__[Flake8]__ invalid escape sequence '\s' ([link](https://sider.review/gh/repos/19784232/pulls/6066#issue-4718578))

<sub>posted by Sider</sub>

Reply:
This warning seems relevant.

Reply:
You're right. I  think Python 3.5.5 which is installed on my elbrus system, doesn't throw a warning about difference of generic and raw strings. I fixed this, unit tests are all passed OK:
```
reisen /usr/src/meson/meson # ./run_unittests.py
pytest-xdist not found, using unittest instead
........s...................s.......s...............................................................................s.....s........../usr/src/meson/meson/tmp6rzjcd61/meson-private/sanitycheckobjc.m: file not recognized: Формат файла не распознан
/usr/src/meson/meson/tmp6rzjcd61/meson-private/sanitycheckobjc.m: file not recognized: Формат файла не распознан
...ss.......s.s..sss............sss.......................................................s.....s........s..s..s..s........s........ssssssssssssssssss
----------------------------------------------------------------------
Ran 283 tests in 1227.003s

OK (skipped=40)
reisen /usr/src/meson/meson #
```
And I hope now everything will be correct even on new versions of Python that deprecated usage of non-standard escape sequences (when old versions of Python do treat these sequences as literals) in standard strings.

By the way, still don't get why vs2015 builds fail. I found that in the build log:
```
[6/6] "C:/Python3/python.exe" "D:/a/1/s/meson.py" "--internal" "exe" "--unpickle" "D:/a/1/s/b 1543608035/meson-private/meson_exe_python.exe_4754b74c4eb41ef8544c125783898d787aaffdd6.dat"
FAILED: out.txt 
"C:/Python3/python.exe" "D:/a/1/s/meson.py" "--internal" "exe" "--unpickle" "D:/a/1/s/b 1543608035/meson-private/meson_exe_python.exe_4754b74c4eb41ef8544c125783898d787aaffdd6.dat"
Traceback (most recent call last):
  File "blaster.py", line 13, in <module>
    import tachyon
ImportError: DLL load failed while importing tachyon: The specified module could not be found.
ninja: build stopped: subcommand failed.
```
But I still don't know what was a reason of that, because I don't know what's here with blaster or tachyon.

Reply:
This is an unrelated CI error that's been happening for everyone.